### PR TITLE
Base Supervisor: initial program outline 

### PR DESCRIPTION
**Pull request by Stadicus**

### CODE REVIEW

File: tools/Makefile [hunk 1]
```diff
@@ -1,2 +1,5 @@
 default:
 	$(MAKE) -C bbbfancontrol
+
+	# uncomment once bbbsupervisor compiles including all dependencies
```
Comment (hkjn) [inline tools/Makefile]: Nit: Maybe we can prefix this comment with `TODO(Stadicus): `?

(It doesn't mean you have to do the work, just that you have the most context for questions about the future work described in the comment.)

File: tools/bbbsupervisor/bbbsupervisor.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/coreos/go-systemd/sdjournal"
+)
+
+//TODO: create "follower" with proper methods (like newFollower, Writer) and data fields
```
Comment (hkjn) [inline tools/bbbsupervisor/bbbsupervisor.go]: Nit: I'd add `TODO(Stadicus)` to these comments as well (see earlier comment).

File: tools/bbbsupervisor/bbbsupervisor.service [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+After=local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/sbin/bbbsupervisor
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
```
Comment (hkjn) [inline tools/bbbsupervisor/bbbsupervisor.service]: Nit: Missing trailing newline at end of file.

**Comment by Stadicus** [comment]
Updated and addressed all nits, thanks!

**Pull request merged**